packages/web/config/ibc-assets.ts

```diff
@@ -1030,6 +1068,46 @@ export const IBCAssetInfos: (IBCAsset & {
     coinMinimalDenom: "ujkl",
     isUnstable: true,
   },
+  {
+    // ALTER
+    counterpartyChainId: "secret-4",
+    sourceChannelId: "channel-476",
+    destChannelId: "channel-44",
+    coinMinimalDenom: "cw20:secret12rcvz0umvk875kd6a803txhtlu7y0pnd73kcej",
+    depositUrlOverride: "https://wrap.scrt.network",
```

Comment:
Please forgive my lack of expertise with Secret. Is wrapping and unwrapping really required for these tokens? The wrapping function appears to be greyed out for all these tokens (and doesn't list stk-scrt at all), and it doesn't look like there is any sTOKEN equivalent for these. It's also interesting the it's only provided for deposit and withdraw...
Also, it seems that this URL is used only for wrapping, but not for transferring between chains.

I'm thinking these deposit URLs should probably be removed....

Reply:
@JeremyParish69 Actually, IBC transfers work fine; you need to connect to Keplr and click on the double arrow next to the icon to IBC asset to Deposit and Withdraw. BAD UX yes this is being worked on to improve with a new page.

Yes, all the other tokens need to be wrapped and unwrapped (the native tokens of other chains). But SHD, ALTR, Sienna, BUTT, and stkd-scrt are tokens (SNIP20s) on the SCRT network, so they are already wrapped and converted directly to IBC (CW20) assets, so NO Wrapping is necessary (and hence greyed out).

stkd-SCRT is missing and needs to be added waiting for an update on that myself.

CC: @assafmo correct me if i am wrong on anything.

Reply:
@mohammedpatla is correct

Reply:
stkd-scrt open to be added to wrap as well https://github.com/scrtlabs/wrap.scrt.network/pull/16

Reply:
Thanks for the explanation.
I have two concerns with this deposit link. The first concern being that, although wrapping and unwrapping might be required for certain functions (such as LPing) for certain tokens, I don't think that this external site is required to IBC these tokens across, as it looks like that can be done directly from the Osmosis site using the normal generated ibc transactions. We want to avoid having to link to external sites wherever possible unless absolutely necessary. So, I'm still thinking that the deposit URL overrides aught to be removed.
The second concern being that if this site truly is required to IBC the tokens around, then I'm thinking it should probably be used for both deposit and withdrawal--not only deposit. (although, I still think the URL override should be removed as long as it's still possible to IBC them around).
If removing the deposit URL override doesn't break the ability to IBC transfer these tokens, then can we please remove them?

Reply:
@assafmo is the only one who can comment on this. I am not entirely sure if this would work without the wrap page

Reply:
To send from Secret you have to send a secret contract tx which involves encryption and custom code in this UI. That's the reason for the override.

To send back from Osmosis it's just like any other IBC token.